docs/whatsnew/1.1.0/1_1_0.rst

```diff
@@ -0,0 +1,55 @@
+Pylake 1.1.0
+============
+
+.. only:: html
+
+Pylake `v1.1.0` has been released. Once again, we'll use this section to highlight some of the new features and an important bugfix.
+
+Global analysis
+---------------
+
+Previously, downstream Kymograph analyses were limited to a single kymograph.
+With global analysis, one can combine tracking results from multiple kymographs to get better statistics when estimating dwell times or calculating diffusion constants.
+For more information on this feature, please refer to the relevant section of the :ref:`kymotracking tutorial<global_analysis>`.
+
+.. figure:: global_analysis.png
+
+    Global analysis allows you to analyze multiple kymographs in a single analysis.
+    This results in improved estimates of the biophysical parameters of your experiment.
+
+Bugfix sample_from_image
```

Comment:
I wonder if we should put this right at the top? I know its' probably more common to put features first and bugfixes later, but I think we should highlight this as much as possible considering the fixed behavior isn't the default. Also, some tick marks would look nicer maybe
```suggestion
Bugfix `KymoTrackGroup.sample_from_image`
```

Reply:
Yeah, we can. I've moved it.